doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -48,7 +48,11 @@ This specification is licensed under the
 
 ## Conformance
 
-All the interfaces described by this specification are mandatory and must be implemented in order to be compliant with. 
+This specification describes the "full" compliance level.
+
+However, next iterations of this specification could define optionality
+in some parts and the compliance level (or roles/profiles)
+corresponding to them.
```

Comment:
I agree in that the specification should mention optionality (and that's the aim of the _"However, next iterations of this specification could define optionality in some parts and the compliance level (or roles/profiles) corresponding to them"_). However, I'm not sure if there is yet a consensus about using `capabilities` field in `/v2` or other alternative mechanism.

All we agree on that?